cep-oci.md

```diff
@@ -96,3 +115,39 @@ Some characters that are used in the conda-forge repository as part of the build
 - `+` is replaced by `__p__`
 - `!` is replaced by `__e__`
 - `=` is replaced by `__eq__`
+
+#### Authentication
+
+Pulling a public image from a Container registry can be done anonymously ([ref](https://docs.github.com/en/packages/learn-github-packages/about-permissions-for-github-packages#visibility-and-access-permissions-for-packages)).
+
+A token can be requested with `pull` scope, using the following URL:
+`https://ghcr.io/token?scope=repository:<org>/<channel-name>/<subdir>/<package-name-or-repodata.json>:pull`
+
+Note that in the case of pulling repodata, the name `repodata.json` is always used in the URL regardless of the encoding.
+
+#### Implementation (conda / mamba / rattler)
+
+##### mamba
+
+In order to fetch packages from an OCI registry, we need to set a mirror (can be more than one) for the channel to be used (e.g `conda-forge`).
+This can be done in the rc file as follows:
+
+```
+mirrored_channels:
+  conda-forge: ["oci://ghcr.io/channel-mirrors/conda-forge"]
+```
+
+When a user requests installing a package (with the configuration set above, and using `conda-forge` channel), a set of requests to fetch `repodata.json` are first performed as follows:
```

Comment:
Why are we marking these as mirrored instead making them first-class things via using the normal channels config setting with an `oci://` prefix to mark OCI channels?

Reply:
I believe this is orthogonal - you should be able to use the `oci://` URL as your only channel as well.

Reply:
Then why do we need the `mirrored_channels` option at all?

Reply:
Maybe not in this CEP? Specifying how we expect channel mirroring works could have merits. Specifically, we should define where the `repodata.json` file comes from when we have multiple mirrors.

Ideally we would also define how we combine download counts from multiple mirrors (but all that would definitely belong into a different CEP).

Reply:
Yeah we should handle mirroring in another CEP I would think.

Reply:
Yes it's orthogonal. I changed it to not mention mirrors anymore.